Sources/Operators.swift

```diff
@@ -204,6 +245,18 @@ public prefix func -(offset: UIOffset) -> UIOffset {
     return UIOffset(horizontal: -offset.horizontal, vertical: -offset.vertical)
 }
 
+public func +(lhs: CGSize, rhs: UIOffset) -> CGSize {
+    return CGSize(width: lhs.width + rhs.horizontal, height: lhs.height + rhs.vertical)
+}
+
+public func +(lhs: UIOffset, rhs: CGSize) -> CGSize {
+    return CGSize(width: rhs.width + lhs.horizontal, height: rhs.height + lhs.vertical)
+}
+
+public func -(lhs: CGSize, rhs: UIOffset) -> CGSize {
+    return CGSize(width: lhs.width - rhs.horizontal, height: lhs.height - rhs.vertical)
+}
+
```

Comment:
Currently I've provided the ability to inset a `CGSize` by a `UIOffset` via the `-(lhs: CGSize, rhs: UIOffset) -> CGSize` operator.

If I allow you to `-(lhs: UIOffset, rhs: CGSize) -> CGSize`, then I'm allowing you to create a `CGSize` by insetting a `UIOffset` with a `CGSize`; that feels like a recipe for unexpected/accidental behavior.